src/QueryBuilder/ExportQueryBuilder.php

```diff
@@ -45,14 +50,26 @@ public function buildQueryFromCommand(
             $this->processFilters($filters, $query, $tableColumnsDefinitions);
         }
 
-        $this->processOrderStatement($orderBy, $query);
-        $this->processSelectStatement(
-            ProtobufHelper::repeatedStringToArray($columns),
-            $query,
-            $tableColumnsDefinitions,
-            $truncateLargeColumns
-        );
-        $this->processFromStatement($schemaName, $tableName, $query);
+        if ($mode === self::MODE_SELECT) {
```

Comment:
Tu by som asi spravil case, kde by boli dve moznosti `select` a `delete` a ak by to nebolo ani jedno tak by som vyhodil exception. Pride mi ze takto by som sa mohol lahko pomylit poslat parameter $mode napr nejaky nezmysel  a ono by mi to zmazalo nieco.

Reply:
OK, to by šlo, phpstan by ti to sice nedovolil, ale kdyby náhodů to někdo někde použil jinde bez něho.

Reply:
upraveno